addons/base_partner_address/static/src/css/base.css

```diff
@@ -1,43 +0,0 @@
-@charset "UTF-8";
```

Comment:
@kaerdsar I added this css file to be able to change the tree column widths for addresses, emails and phones in the partner detail view. Did you remove the file accidently?